src/main/java/CalendarCommand.java

```diff
@@ -0,0 +1,30 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CalendarCommand extends Command {
```

Comment:
I think this "Calendar" command is a great idea! I'd definitely use it to get a glance of all my tasks that involve a certain date time.